aospy/calc.py

```diff
@@ -254,7 +253,7 @@ def _get_input_data_paths_gfdl_dir_struct(self, name, data_in_direc,
                                                   self.intvl_out,
                                                   self.data_in_start_date[n].year,
                                                   self.data_in_dur[n]))
-                 for year in range(start_year, end_year + 1)]
```

Comment:
Why is the `+1` removed?  Seems like without it you won't get the `end_year` included.